controllers/bounce_processes.go

```diff
@@ -181,28 +181,18 @@ func getProcessesReadyForRestart(ctx context.Context, logger logr.Logger, r *Fou
 
 		addresses = append(addresses, addressMap[process]...)
 
-		processGroupID := podmanager.GetProcessGroupIDFromProcessID(process)
-		pod, err := r.PodLifecycleManager.GetPods(ctx, r, cluster, internal.GetSinglePodListOptions(cluster, processGroupID)...)
-		if err != nil {
-			return nil, &requeue{curError: err}
-		}
-		if len(pod) == 0 {
-			return nil, &requeue{message: fmt.Sprintf("No pod defined for process group ID: \"%s\"", processGroupID), delay: podSchedulingDelayDuration}
-		}
-
-		synced, err := r.updatePodDynamicConf(cluster, pod[0])
```

Comment:
I thought you said during the review session that "update_status" controller already calls "updatePodDynamicConf()" and so we don't need to call it here - is that correct? But I see "update_config_pod" reconciler calling this method, and not "update_status"? Anyway, we don't need to call this here because "update_config_pod" reconciler is already calling it on each pod?

Reply:
What I said in the review session was that in the update_status we set the `fdbv1beta2.IncorrectConfigMap` and the actual call of updatePodDynamicConf will happen in `update_config_pod`. I probably was a little bit quick going over those two files.